tests/test_tools.py

```diff
@@ -0,0 +1,117 @@
+import pytest
+import logging
+from pkg_resources import resource_filename
+from wps_tools.utils import (
+    is_opendap_url,
+    get_filepaths,
+    collect_output_files,
+    build_meta_link,
+    log_handler,
+)
+from wps_tools.testing import (
+    local_path,
+    opendap_path,
+    run_wps_process,
+)
+from processes.wps_say_hello import SayHello
+
+nc_file = "gdd_annual_CanESM2_rcp85_r1i1p1_1951-2100.nc"
+
+
+class NCInput:  # For testing 'get_filepaths'
+    def __init__(self, url="", file=""):
+        self.url = url
+        self.file = file
+
+
+class Response:  # For testing 'log_handler'
+    def __init__(self):
+        self.message = ""
+        self.status_percentage = 0
+
+    def update_status(self, message, status_percentage):
+        self.message = message
+        self.status_percentage = status_percentage
+
+
+# Test 'testing' functions
+def test_local_path():
+    assert f"tests/data/{nc_file}" in local_path(nc_file)
+
+
+def test_opendap_path():
+    assert f"datasets/TestData/{nc_file}" in opendap_path(nc_file)
+
+
+def test_run_wps_process():
+    params = "name=PCIC"
+    run_wps_process(SayHello(), params)
```

Comment:
Even though there is only a single case can we use `pytest.mark.parametrize` here? It will make adding more cases in the future a little bit easier.